.github/workflows/main.yml

```diff
@@ -0,0 +1,29 @@
+
+name: Backend workflow for Golang using Snyk
+
+on:
+  push:
```

Comment:
I think the job should run on pull request, this can be removed

Reply:
okay